src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -251,13 +269,60 @@ def data_metrics(
                     WIDTH_UUID: widths[i],
                     AUC_UUID: auc_per_twitch[i],
                     TWITCH_FREQUENCY_UUID: twitch_frequencies[i],
+                    CONTRACTION_VELOCITY_UUID: contraction_velocity[i],
+                    RELAXATION_VELOCITY_UUID: relaxation_velocity[i],
                 }
             }
         )
 
     return main_twitch_dict, aggregate_dict
 
 
+def calculate_twitch_velocity(
+    twitch_indices: NDArray[int],
+    widths: List[Dict[int, Dict[UUID, Union[Tuple[int, int], int]]]],
+    is_contraction: bool,
+) -> NDArray[Any]:
```

Comment:
not sure if this causes an issue with mypy or other typing, but can you specify the NDArray type here? is it `float`?

Reply:
--instead of just `Any`